en/gps_compass/rtk_gps.md

```diff
@@ -140,6 +161,89 @@ See the [EKF2 GPS Configuration](../advanced_config/tuning_the_ecl_ekf.md#gps) s
+* [mRo u-Blox ZED-F9 RTK L1/L2 GPS](https://store.mrobotics.io/product-p/mr-m10020-a.htm) (store.mrobotics.io)
+
+:::note
+[Freefly RTK GPS](../gps_compass/rtk_gps_freefly.md) and [Holybro H-RTK F9P GNSS](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md) cannot be used because they do not expose the UART2 port.
+:::
+
+
+
+### Configuration/Setup
+
+In overview:
+- The UART2 of the GPS devices need to be connected together (TXD2 of the "Moving Base" to RXD2 of the "Rover")
+- Connect UART1 on each of the GPS to (separate) unused UART's on the autopilot, and configure both of them as GPS with baudrate set to `Auto`.
+  The mapping is as follows:
+  - Main GPS = Rover
+  - Secondary GPS = Moving Base
+- Set [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `Heading` (1)
+- Reboot and wait until both devices have GPS reception.
+  `gps status` should then show the Main GPS going into RTK mode, which means the heading angle is available.
+
+:::note
+If using RTK with a fixed base station the secondary GPS will show the RTK state w.r.t. the base station.
+:::
+
+Ideally the two antennas should be identical, on the same level/horizontal plane and oriented the same way, and on an identical ground plane size and shape ([Application note](https://www.u-blox.com/sites/default/files/ZED-F9P-MovingBase_AppNote_%28UBX-19009093%29.pdf), section *System Level Considerations*)). 
+
+Otherwise the antennas can be positioned as needed, but the [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) must be configured.
+This offset is the angle made by the *baseline* (the line between the two GPS modules) relative to the vehicle x-axis (front/back axis, as shown [here](../config/flight_controller_orientation.md#calculating-orientation)).
+
+:::note
+The application note does not state the minimal required separation between modules.
+50cm has been used in test vehicles running PX4.
+:::
+
+To activate heading fusion for the attitude estimation, set the [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) parameter to enable *GPS yaw fusion*.
+
+
+## RTK GPS that Output Yaw
+
+Some RTK GPS units have multiple antennas, and can output a yaw angle.
+This yaw can be used instead of the heading from the magnetic compass (and as an alternative to [RTK Heading with Two FP9 modules](#rtk-gps-heading-setup-configuration-u-blox-f9p)).
+
+###  Supported Devices
+
+* [Trimble MB-Two](../gps_compass/rtk_gps_trimble_mb_two.md)
+
+### Configuration
+
+[RTK Positioning](#positioning-setup-configuration) should be set up as normal.
+
+In order to use the GPS for yaw fusion you will also need to configure:
+
+Parameter | Setting
+--- | ---
+[GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) |  The angle made by the *baseline* (the line between the two GPS antennas) relative to the vehicle x-axis (front/back axis, as shown [here](../config/flight_controller_orientation.md#calculating-orientation)).
```

Comment:
@bkueng Is this correct? i.e. I have assumed that the offset is calculated relative to baseline in exaclyt the same way as for F9P

Reply:
Yes, same meaning